src/main/java/uk/gov/hmcts/darts/audit/controller/AuditController.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.hmcts.darts.audit.controller;
+
+import org.springframework.http.ResponseEntity;
+import uk.gov.hmcts.darts.audit.api.AuditApi;
+import uk.gov.hmcts.darts.audit.model.SearchResult;
+
+import java.time.OffsetDateTime;
+
+public class AuditController implements AuditApi {
+    @Override
+    public ResponseEntity<SearchResult> search(Object startDate, OffsetDateTime endDate, String caseId, String eventId) {
```

Comment:
This needs to be regenerated again after the OAS tweaks to correct `startDate`

Reply:
ACCEPT